orchagent/neighorch.cpp

```diff
@@ -671,19 +671,6 @@ void NeighOrch::doTask(Consumer &consumer)
 
         IpAddress ip_address(key.substr(found+1));
 
-        /* Verify Ipv4 LinkLocal and skip neighbor entry added for RFC5549 */
-        if ((ip_address.getAddrScope() == IpAddress::LINK_SCOPE) && (ip_address.isV4()))
-        {
-            /* Check if this prefix is not a configured ip, if so allow */
-            IpPrefix ipll_prefix(ip_address.getV4Addr(), 16);
-            if (!m_intfsOrch->isPrefixSubnet (ipll_prefix, alias))
-            {
-                SWSS_LOG_NOTICE("Skip IPv4LL neighbor %s, Intf:%s op: %s ", ip_address.to_string().c_str(), alias.c_str(), op.c_str());
-                it = consumer.m_toSync.erase(it);
-                continue;
-            }
-        }
-
         NeighborEntry neighbor_entry = { ip_address, alias };
```

Comment:
root@sonic:/home/admin# redis-cli hgetall "NEIGH_TABLE:Ethernet0:169.254.0.1"
1) "family"
2) "IPv4"
3) "neigh"
4) "00:10:94:00:00:01"

Is this static or dynamic neighbor?

Reply:
For testing, the neighbour/route entries were manually added to APP DB. In typical use-case, this neighbour entry will be static.

Reply:
Okay, which module would push this static ARP and MAC into the kernel? I believe, we should push static nbr entry add/del code as well to community to support BGP unnumbered use-case, right?

Reply:
FRR(Zebra) is already adding static(PERMANENT) IPv4 LL neighbours in kernel. Please check the APP DB log in the issue. This change is only to allow such entries in NeighOrch as well.

Reply:
@utpalkantpintoo Can you attach the testing logs of BGP unnumbered use-case with the community image, just to make sure this code is working fine for that use-case?